src/main/resources/db/changelog/changes/231120241608-create-shopping-cart-table.yaml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
```

Comment:
This file name starts with number with a lot of digits, why?

Reply:
it's just common best practice... And how do you handle migrations in your commercial project?..

Reply:
is it timestamp?

Reply:
I'd say it's a bad practice :)

Reply:
Okay, but this is an article from Baeldung, not official documentation or anything like that. What if we find another article that says this is bad practice and you should do it differently. Which one of them are we going to believe?)

Reply:
Then we'll ask ChatGPT)
Joking aside, every solution has its advantages and disadvantages. I used this approach on commercial project and continue to use it here in educational project. It's my choice